File: Android/MainActivity.java
package com.example.helloworld;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.widget.Button;
import android.view.View;
import android.content.Intent;

public class MainActivity extends  AppCompatActivity implements View.OnClickListener
{

    private Button buttonStartReceiving;
    private boolean end = false;

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        buttonStartReceiving = (Button) findViewById(R.id.startRecieving);

        buttonStartReceiving.setOnClickListener(this);
    }

    @Override
    public void onClick(View v)
    {
        if (v.getId() == R.id.startRecieving) {
             Log.i("Button pressed. ", "Starting to listen...");

             Intent intent = new Intent(this, ClientActivity.class);
             startActivity(intent);
        }
    }
}

File: Android/Kotlin/EulersPhi/MainActivity.kt
package com.example.eulersphi

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val startButton = findViewById<Button>(R.id.start)

        startButton.setOnClickListener {
            val intent = Intent(this, EulersPhiActivity::class.java)
            startActivity(intent)
        }
    }
}